packages/duckdb-server/package.json

```diff
@@ -3,8 +3,10 @@
   "version": "0.11.0",
   "private": true,
   "scripts": {
-    "prepublishOnly": "rimraf dist && mkdir dist && hatch run test:cov && hatch fmt --check && hatch build",
-    "publish": "hatch publish --user __token__",
-    "release": "npm run prepublishOnly && npm run publish"
+    "prepublishOnly": "uv run pytest && uv run ruff check && uv run ruff format --check",
+    "publish": "uv build && uvx twine upload --skip-existing dist/*",
```

Comment:
Don't mind this at all! We do something similar in anywidget. The funny thing is that uv is so flexible that you could probably publish a package to PyPI that just reads scripts from somewhere in the `pyproject.toml` and calls it (api that doesn't exist):

```
uvx project-scripts <script-name>
```

Reply:
Haha, that's a nice hack. I hope uv will support scripts/shortcuts/commands/dev-scripts (whatever they will call it) eventually to clean this up.

Reply:
Agreed. IIRC, there has been some discussion around this, but I don't remember the thread or where I saw it.